packages/flutter_tools/lib/src/commands/build_windows.dart

```diff
@@ -46,16 +59,21 @@ class BuildWindowsCommand extends BuildSubCommand {
   Future<FlutterCommandResult> runCommand() async {
     final FlutterProject flutterProject = FlutterProject.current();
     final BuildInfo buildInfo = await getBuildInfo();
+    // Target same platform as the host one.
```

Comment:
Could you clarify this comment? It sounds like it's saying the host platform is used as the target platform.

Reply:
It means that when you run `flutter run` on an x64 machine, it builds and run an x64 version, and on arm64, an arm64 version.

This comment seems to create more confusion that help, and a reader can see that default value of target-platform is depending on host, so I'll just remove it.